### PR TITLE
Some improvements and implementations

### PR DESCRIPTION
**Pull request by williamdes**
Ref: #238
Closes: #240
Closes: #239
Closes: #233
Closes: #189
Closes: #139

Duplicate of: #241 that was reverted after a merge mistake

### CODE REVIEW

**Comment by williamdes** [comment]
```bash
docker build ./apache/ -t apache-test
docker run --rm --net=local_williamdeslocal -p 8080:80 -e PMA_ARBITRARY=1 -eUPLOAD_LIMIT=4G -it apache-test:latest apache2-foreground
```
image

File: apache/docker-entrypoint.sh [hunk 1]
```diff
@@ -16,9 +16,9 @@ if [[ "$1" == apache2* ]] || [ "$1" == php-fpm ]; then
         group="$(id -g)"
     fi
 
-    chown www-data:www-data /sessions /var/nginx/client_body_temp
+    chown ${user}:${group} /sessions /var/nginx/client_body_temp
```
Comment (williamdes) [inline apache/docker-entrypoint.sh]: @J0WI That would make sense, do you agree ?

**Comment by williamdes** [comment]
@J0WI can I request your review on this PR ?

File: apache/docker-entrypoint.sh (hunk 1 above)
Comment (J0WI) [reply apache/docker-entrypoint.sh]: I think these directories are not longer used.

File: docker-entrypoint.sh [hunk 2]
```diff
@@ -47,4 +47,15 @@ EOT
     fi
 fi
 
+UPLOAD_LIMIT_INI_FILE="$PHP_INI_DIR/conf.d/phpmyadmin-upload-limit.ini"
+if [ ! -z "${UPLOAD_LIMIT}" ]; then
+    echo "Adding the custom upload limit."
+    echo -e "upload_max_filesize = $UPLOAD_LIMIT\npost_max_size = $UPLOAD_LIMIT\n" > $UPLOAD_LIMIT_INI_FILE
+else
+    if [ -f $UPLOAD_LIMIT_INI_FILE ]; then
```
Comment (J0WI) [inline docker-entrypoint.sh]: In which case is `$UPLOAD_LIMIT_INI_FILE` present without specifying a `$UPLOAD_LIMIT`?

File: apache/docker-entrypoint.sh (hunk 1 above)
Comment (williamdes) [reply apache/docker-entrypoint.sh]: Makes sense, will remove the line

Comment (J0WI) [reply apache/docker-entrypoint.sh]: There are a few other references to them. The session can still be customized by mounting a specific config.

Comment (williamdes) [reply apache/docker-entrypoint.sh]: Ah, I will research and try to add a comment

File: docker-entrypoint.sh (hunk 2 above)
Comment (williamdes) [reply docker-entrypoint.sh]: I need to test by maybe with docker compose when changing the value

Comment (williamdes) [reply docker-entrypoint.sh]: c4af17e703dfcce8b698cf6b6c90f287bdcc3f0e

**Comment by CaliforniaMountainSnake** [comment]
Is this available in 5.1?

**Comment by williamdes** [comment]
> Is this available in 5.1?

Sure, just have a look at the readme of this repository

**Pull request merged**